app/js/app/router.js

```diff
@@ -103,122 +107,144 @@ define(["angular-ui-router"], function() {
-            .state('physics_skills_14', staticPageState("/physics_skills_14", "book_index" , "BooksControllers"))
+            .state('physics_skills_14', staticPageState("/physics_skills_14", "book_index", "BooksControllers"))
             .state('questions', staticPageState('/questions', 'questions', 'QuestionsPageControllers'))
             .state('publications', genericPageState("/publications", "publications"))
             .state('prize_draws', genericPageState("/prize_draws", "prize_draws"))
             .state('spc', genericPageState("/spc", "spc"))
-            
-            .state('teacher_features', {
-                url:"/teacher_features?redirectModal",
-                views: {
-                    "body": {
-                        templateUrl: "/partials/states/teacher_features.html",
-                        controller: "TeacherFeaturesPageController",
-                    }
-                },
-                reloadOnSearch: false,
-            })
 
-            .state('equality', {
-                url: "/equality?symbols",
-                resolve: {
-                    // BIG RED AND YELLOW WARNING WITH SPARKLES AND A FEW CRACKERS JUST IN CASE:
-                    // we may want to revert this policy at some point.
-                    // requireRole: getRolePromiseInjectableFunction(["ADMIN", "CONTENT_EDITOR", "EVENT_MANAGER"]),
-                },
-                views: {
-                    "body": { 
-                        templateUrl: "/partials/states/equation_editor.html",
-                        controller: ["$scope", '$stateParams', function($scope, $stateParams) {
-                            if ($stateParams.symbols) {
-                                $scope.questionDoc = {availableSymbols: $stateParams.symbols.split(",")};
-                            }
-                            $scope.selectedFormula = {
-                                symbols: {},
+        .state('teacher_features', {
+            url: "/teacher_features?redirectModal",
+            views: {
+                "body": {
+                    templateUrl: "/partials/states/teacher_features.html",
+                    controller: "TeacherFeaturesPageController",
+                }
+            },
+            reloadOnSearch: false,
+        })
+
+        .state('equality', {
+            url: "/equality?mode&symbols",
+            resolve: {
+                // BIG RED AND YELLOW WARNING WITH SPARKLES AND A FEW CRACKERS JUST IN CASE:
+                // we may want to revert this policy at some point.
+                // requireRole: getRolePromiseInjectableFunction(["ADMIN", "CONTENT_EDITOR", "EVENT_MANAGER"]),
+            },
+            views: {
+                "body": {
+                    templateUrl: "/partials/states/equation_editor.html",
+
+                    controller: ['$scope', '$rootScope', '$stateParams', function($scope, $rootScope, $stateParams) {
```

Comment:
Could we think about moving this to it's own file? It's getting a bit long to be inline now.

`EqualityPageController.js` perhaps, in with the others?